packages/smooth_app/lib/pages/product/product_page.dart

```diff
@@ -280,6 +280,55 @@ class _ProductPageState extends State<ProductPage> {
     localDatabase.notifyListeners();
   }
 
+  Widget _buildProductImagesCarousel(BuildContext context) {
+    final List<ImageUploadCard> carouselItems = <ImageUploadCard>[
+      ImageUploadCard(
+          product: _product,
+          imageField: ImageField.FRONT,
+          imageUrl: _product.imageFrontUrl,
+          title: 'Product',
+          buttonText: 'Front photo'),
+      ImageUploadCard(
+          product: _product,
+          imageField: ImageField.INGREDIENTS,
+          imageUrl: _product.imageIngredientsUrl,
+          title: 'Ingredients',
+          buttonText: 'Ingredients photo'),
+      ImageUploadCard(
+        product: _product,
+        imageField: ImageField.NUTRITION,
+        imageUrl: _product.imageNutritionUrl,
+        title: 'Nutrition',
+        buttonText: 'Nutrition facts photo',
+      ),
+      ImageUploadCard(
+        product: _product,
+        imageField: ImageField.PACKAGING,
+        imageUrl: _product.imagePackagingUrl,
+        title: 'Packaging information',
+        buttonText: 'Packaging information photo',
+      ),
+      ImageUploadCard(
+        product: _product,
+        imageField: ImageField.OTHER,
+        imageUrl: null,
+        title: 'More photos',
+        buttonText: 'More photos',
+      ),
+    ];
+
+    return CarouselSlider(
+        options: CarouselOptions(
+          disableCenter: true,
+          enableInfiniteScroll: false,
+          height: 200, // Small images are 200px high, normal images are 400px
```

Comment:
One day we'll have to be more consistent regarding the sizes, perhaps with a "size repository": once we use a percentage of the screen size, once we use a given float size (which may mean pixels or dpi).
But maybe not today ;)

Reply:
Sure, we'll have to experiment to see what works best on different devices.